src/utils/Utils.ts

```diff
@@ -197,55 +197,54 @@ export class Utils {
   }
 
   /**
-   * Get the value of a field.
-   * If multiple names are provided the first result not null is returned.
+   * Get the value of the first field from the array and defined in the result.
    *
    * @param result a QueryResult in which to ge the fieldvalue.
    * @param name One or multiple fieldNames to get the value.
    */
-  static getFieldValue(result: IQueryResult, name: string | Array<string>): any {
-    if (Array.isArray(name)) {
-      for (let i = 0; i < name.length; i++) {
-        let value = this.getFieldValue(result, name[i]);
-        if (value !== undefined) {
-          return value;
-        }
+  static getFirstAvailableFieldValue(result: IQueryResult, fieldNames: Array<string>): string | undefined {
```

Comment:
While both are similar ( `string[]` versus `Array<string>` ) it would be better to use `string[]` for uniformity reasons.